src/src/PluginStructs/P128_data_struct.cpp

```diff
@@ -363,28 +363,50 @@ bool P128_data_struct::plugin_write(struct EventStruct *event,
 
       _counter_mode_step = 0;
 
+      hex2rrggbb("000000");
```

Comment:
Please use F() macro here:
```c++
hex2rrggbb(F("000000"));

Reply:
I copy paste this, but I can replace all occurrences, no problem